Formula/cartesi-machine-rootfs-image.rb

```diff
@@ -0,0 +1,18 @@
+class CartesiMachineRootfsImage < Formula
+  desc "Rootfs image for the Cartesi Machine"
+  homepage "https://cartesi.io/"
+  VERSION = "0.16.1".freeze
+  ROOTFS_IMAGE = "rootfs-tools-v#{VERSION}.ext2".freeze
+  url "https://github.com/cartesi/machine-emulator-tools/releases/download/v#{VERSION}/#{ROOTFS_IMAGE}"
+  sha256 "4db885fdb4f013922d8ea8474768148ac4d45460a4ef30aea823836ea72ffed9"
+  license "Apache-2.0"
```

Comment:
```suggestion
  license "GPL-2.0"
```

Reply:
Apache-2.0 is the build script only, same reasoning as before. This is actually a lot of things, including GPL-2.0, more something like https://github.com/cartesi/machine-guest-tools/releases/download/v0.17.0-test2/rootfs-tools-v0.17.0-test2.ext2.html
But I don't know how to place multiple licenses in a simple file like this.